src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,23 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { MovieCard } from '../MovieCard/MovieCard';
+
 import './MoviesList.scss';
 
-export const MoviesList = () => (
-  <>Put the list here</>
+export const MoviesList = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <MovieCard {...movie} key={movie.imdbId} />
+    ))}
+  </div>
+);
+
+const moviesType = PropTypes.arrayOf(
```

Comment:
Називай типи з великої букви
```suggestion
const MoviesType = PropTypes.arrayOf(
```